src/components/Images/MyGallery.vue

```diff
@@ -40,43 +48,58 @@ const getThumbnails = async (sessionId) => {
       if (data.results.length > 0) {
         thumbnailsMap.value[sessionId] = data.results.map(result => result.url)
       }
-      loading.value = false
+      checkIfLoadingComplete()
```

Comment:
why was this needed?

Reply:
I was testing something out cause my tests weren't passing but it was not related and forgot to update!